spec/spec_helper.rb

```diff
@@ -38,6 +39,7 @@
   config.infer_spec_type_from_file_location!
   config.use_transactional_fixtures = true
   config.include FactoryBot::Syntax::Methods
+  config.include GdsApi::TestHelpers::PublishingApi
```

Comment:
Oh boo. Is this only in v1 test helpers? that's a pain.

Reply:
Yep, unfortunately! I mean I could include it directly in the test file itself but as I still need to do the destroying intent workflow when someone cancels a scheduling it makes sense to have it here as I will need the helper methods for that piece of work too.